ui/src/Constant/constant.js

```diff
@@ -1,5 +1,13 @@
 export const FOOTER = {
-    "COPYRIGHT": "Copyright 2022-2023",
-    "RDS_WEBSITE": "RDS Website",
-    "DISCORD_SERVER":"Discord server"
-}
\ No newline at end of file
+  COPYRIGHT: "Copyright 2022-2023",
+  RDS_WEBSITE: "RDS Website",
+  DISCORD_SERVER: "Discord server",
+};
+
+export const TYPE_TO_COLOR = {
```

Comment:
What is `TYPE_TO_COLOR`?

Reply:
basically an object to map the type of notification to a color

Reply:
The name should be `NOTIFICATION_TEXT_COLOR` wyt?